src/pouchdb/sendDbByEmail.ts

```diff
@@ -0,0 +1,146 @@
+
+    const subject = `DB files for ${instance}`
+
+    const files = await RNFS.readDir(RNFS.DocumentDirectoryPath)
+
+    const dbFiles = files.filter(f => f.name.startsWith(`${fqdn}_`))
+
+    const externalFiles = []
+    for (const dbFile of dbFiles) {
+      const dirs = RNFetchBlob.fs.dirs
+
+      const internalPath = dbFile.path
+
+      if (Platform.OS === 'android') {
+        const date = Number(new Date())
+        const externalPath = `${dirs.DCIMDir}/DbFile_${dbFile.name}${date}.sqlite`
+
+        await RNFS.copyFile(internalPath, externalPath)
+
+        externalFiles.push({
+          path: externalPath
+        })
+      } else {
+        externalFiles.push({
+          path: dbFile.path,
+          type: 'pdf' // there is no compatible MIME type, so we use PDF one as replacement, this should change nothing expect the email aspect
+        })
+      }
+    }
+
+    await showSplashScreen(splashScreens.SEND_LOG_EMAIL)
+    log.info('Start email intent', externalFiles)
+    await sendMailPromise(subject, supportEmail, externalFiles).catch(
+      (errorData: sendMailError) => {
+        const { error, event } = errorData
+        Alert.alert(
+          error,
+          event,
+          [
+            {
+              text: 'Ok',
+              onPress: (): void => log.debug('OK: Email Error Response')
+            },
+            {
+              text: 'Cancel',
+              onPress: (): void => log.debug('CANCEL: Email Error Response')
+            }
+          ],
+          { cancelable: true }
+        )
+      }
+    )
+    log.info('Did finish email intent')
+    await hideSplashScreen(splashScreens.SEND_LOG_EMAIL)
+  } catch (error) {
+    const errorMessage = getErrorMessage(error)
+    log.error('Error while trying to send DB email', errorMessage)
+  }
+}
+
+const sendMailPromise = (
+  subject: string,
+  email: string,
+  attachments: Attachment[]
+): Promise<void> => {
+  return new Promise((resolve, reject) => {
+    Mailer.mail(
+      {
+        subject: subject,
+        recipients: ['yannick.chiron@gmail.com'],//email],
+        body: buildMessageBody(),
+        isHTML: true,
+        attachments: attachments
+      },
+      (error, event) => {
+        if (error) {
+          reject({ error, event })
+        } else {
+          resolve()
+        }
+      }
+    )
+  })
+}
+
+const buildMessageBody = (): string => {
+  const appVersion = DeviceInfo.getVersion()
+  const appBuild = DeviceInfo.getBuildNumber()
+  const bundle = DeviceInfo.getBundleId()
+  const deviceBrand = DeviceInfo.getBrand()
+  const deviceModel = DeviceInfo.getModel()
+  const os = DeviceInfo.getSystemName()
+  const version = DeviceInfo.getSystemVersion()
+
+  const appInfo = `App info: ${appVersion} (${appBuild})`
+  const bundleInfo = `App bundle: ${bundle}`
+  const deviceInfo = `Device info: ${deviceBrand} ${deviceModel} ${os} ${version}`
+
+  return `${appInfo}\n${bundleInfo}\n${deviceInfo}`
+}
```

Comment:
Same method as [src/app/domain/logger/fileLogger.ts](https://github.com/cozy/cozy-flagship-app/blob/be4e45832f7a83b13ef46d1686e735b0ec9a001b/src/app/domain/logger/fileLogger.ts#L101)

Reply:
We can yes, for now it is not really important, those are debug methods so it won't be problematic if they differ someday